master/buildbot/statistics/storage_backends/influxdb_client.py

```diff
@@ -58,17 +47,20 @@ def __init__(self, url, port, user, password, db, captures,
                                      self.password, self.db)
         self.inited = True
```

Comment:
it is not recommended to play too much with name mangling :) self._inited or self.inited is ok for me